aries-site/src/data/wcag/components.json

```diff
@@ -47,6 +47,402 @@
       { "rule": "3.2.4", "status": "conditional" },
       { "rule": "3.2.5", "status": "passed" },
       { "rule": "4.1.2", "status": "passed" }
+    ],
+  "avatar": [
+      {
+        "rule": "1.1.1",
+        "label": "Non-text content",
+        "status": "conditional",
+        "notes": "Requires caller to provide a11yTile prop for `aria-label` to be added to the element for screen readers"
+      },
+      {
+        "rule": "1.2",
+        "label": "Time-based Media",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.3.1",
+        "label": "Info and Relationships",
+        "status": "passed",
+        "notes": "Uses `role='figure'` for avatar container and `role='presentation'` for the image element."
+      },
+      {
+        "rule": "1.3.2",
+        "label": "Meaningful Sequence",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.3.3",
+        "label": "Sensory Characteristics",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.3.4",
+        "label": "Orientation",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.3.5",
+        "label": "Identify Input Purpose",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.3.6",
+        "label": "Identify Purpose",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.4.1",
+        "label": "Use of Color",
+        "status": "passed",
```

Comment:
This is conditional based on what background color is passed to the avatar. By default there is not a ackgroud color